examples/serverless/index.ts

```diff
@@ -15,10 +15,11 @@
 import * as gcp from "@pulumi/gcp";
 
 // Create the Function resource
-let f = new gcp.serverless.Function("f", {}, 
-    (req: any, res: any) => {
+let f = new gcp.cloudfunctions.HttpCallbackFunction("f", {
+    callback: (req: any, res: any) => {
         res.send(`Hello ${req.body.name || 'World'}!`);
-    });
+    },
```

Comment:
yes.  we could also have an overload for that.  will change.  We def need to allow args though in case you want to supply overrides for things like memory utilization.

Reply:
Definitely agree we need the longfrom as well.

Reply:
both sort and long forms are now supported!